From 59742d26541b526c1185dadab368e1ef3ab85a1e Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Tue, 19 Aug 2014 15:22:55 -0400
Subject: [x86] perf/intel: Clean up EVENT_ATTR_STR() muck

Message-id: <1408462094-14194-7-git-send-email-jolsa@redhat.com>
Patchwork-id: 87993
O-Subject: [PATCH RHEL7.1 BZ1131394 006/325] perf/x86/intel: Clean up EVENT_ATTR_STR() muck
Bugzilla: 1131394
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1131394
https://bugzilla.redhat.com/show_bug.cgi?id=1131394

upstream
========
commit 7f2ee91f54fb56071f97bde1ef7ba7ba0d58dfe5
Author: Ingo Molnar <mingo@kernel.org>
Date: Thu Sep 12 19:17:00 2013 +0200

description
===========
Make the code a bit more readable by removing stray whitespaces et al.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/x86/kernel/cpu/perf_event_intel.c b/arch/x86/kernel/cpu/perf_event_intel.c
index 3e032c0..8420041 100644
--- a/arch/x86/kernel/cpu/perf_event_intel.c
+++ b/arch/x86/kernel/cpu/perf_event_intel.c
@@ -189,9 +189,9 @@ static struct extra_reg intel_snbep_extra_regs[] __read_mostly = {
  EVENT_EXTRA_END
 };
 
-EVENT_ATTR_STR(mem-loads, mem_ld_nhm, "event=0x0b,umask=0x10,ldlat=3");
-EVENT_ATTR_STR(mem-loads, mem_ld_snb, "event=0xcd,umask=0x1,ldlat=3");
-EVENT_ATTR_STR(mem-stores, mem_st_snb, "event=0xcd,umask=0x2");
+EVENT_ATTR_STR(mem-loads, mem_ld_nhm, "event=0x0b,umask=0x10,ldlat=3");
+EVENT_ATTR_STR(mem-loads, mem_ld_snb, "event=0xcd,umask=0x1,ldlat=3");
+EVENT_ATTR_STR(mem-stores, mem_st_snb, "event=0xcd,umask=0x2");
 
 struct attribute *nhm_events_attrs[] = {
  EVENT_PTR(mem_ld_nhm),
@@ -2218,23 +2218,22 @@ static __init void intel_nehalem_quirk(void)
  }
 }
 
-EVENT_ATTR_STR(mem-loads,      mem_ld_hsw,     "event=0xcd,umask=0x1,ldlat=3");
-EVENT_ATTR_STR(mem-stores,     mem_st_hsw,     "event=0xd0,umask=0x82")
+EVENT_ATTR_STR(mem-loads, mem_ld_hsw, "event=0xcd,umask=0x1,ldlat=3");
+EVENT_ATTR_STR(mem-stores, mem_st_hsw, "event=0xd0,umask=0x82")
 
 /* Haswell special events */
-EVENT_ATTR_STR(tx-start,        tx_start,       "event=0xc9,umask=0x1");
-EVENT_ATTR_STR(tx-commit,       tx_commit,      "event=0xc9,umask=0x2");
-EVENT_ATTR_STR(tx-abort,        tx_abort, "event=0xc9,umask=0x4");
-EVENT_ATTR_STR(tx-capacity,     tx_capacity, "event=0x54,umask=0x2");
-EVENT_ATTR_STR(tx-conflict,     tx_conflict, "event=0x54,umask=0x1");
-EVENT_ATTR_STR(el-start,        el_start,       "event=0xc8,umask=0x1");
-EVENT_ATTR_STR(el-commit,       el_commit,      "event=0xc8,umask=0x2");
-EVENT_ATTR_STR(el-abort,        el_abort, "event=0xc8,umask=0x4");
-EVENT_ATTR_STR(el-capacity,     el_capacity,    "event=0x54,umask=0x2");
-EVENT_ATTR_STR(el-conflict,     el_conflict,    "event=0x54,umask=0x1");
-EVENT_ATTR_STR(cycles-t,        cycles_t,       "event=0x3c,in_tx=1");
-EVENT_ATTR_STR(cycles-ct,       cycles_ct,
-     "event=0x3c,in_tx=1,in_tx_cp=1");
+EVENT_ATTR_STR(tx-start, tx_start, "event=0xc9,umask=0x1");
+EVENT_ATTR_STR(tx-commit, tx_commit, "event=0xc9,umask=0x2");
+EVENT_ATTR_STR(tx-abort, tx_abort, "event=0xc9,umask=0x4");
+EVENT_ATTR_STR(tx-capacity, tx_capacity, "event=0x54,umask=0x2");
+EVENT_ATTR_STR(tx-conflict, tx_conflict, "event=0x54,umask=0x1");
+EVENT_ATTR_STR(el-start, el_start, "event=0xc8,umask=0x1");
+EVENT_ATTR_STR(el-commit, el_commit, "event=0xc8,umask=0x2");
+EVENT_ATTR_STR(el-abort, el_abort, "event=0xc8,umask=0x4");
+EVENT_ATTR_STR(el-capacity, el_capacity, "event=0x54,umask=0x2");
+EVENT_ATTR_STR(el-conflict, el_conflict, "event=0x54,umask=0x1");
+EVENT_ATTR_STR(cycles-t, cycles_t, "event=0x3c,in_tx=1");
+EVENT_ATTR_STR(cycles-ct, cycles_ct, "event=0x3c,in_tx=1,in_tx_cp=1");
 
 static struct attribute *hsw_events_attrs[] = {
  EVENT_PTR(tx_start),
-- 
1.7.1